hoprd/rest-api/src/lib.rs

```diff
@@ -1386,6 +1469,121 @@ mod messages {
         }
     }
 
+    #[derive(Debug, Default, Clone, serde::Deserialize, utoipa::ToSchema)]
+    #[schema(value_type = String)] //, format = Binary)]
+    pub struct Text(String);
+
+    #[derive(Debug, Clone, serde::Deserialize)]
+    pub(crate) struct WebSocketSendMsg {
+        pub cmd: String,
+        pub args: SendMessageReq,
+    }
+
+    #[derive(Debug, Clone, serde::Serialize)]
+    pub(crate) struct WebSocketReadMsg {
+        #[serde(rename = "type")]
+        type_: String,
+        pub tag: u16,
+        pub body: String,
+    }
```

Comment:
Are the WS messages also part of the OpenAPI schema, or it does not apply ?

Reply:
No, they cannot be described in an OpenAPI schema. Only HTTP.